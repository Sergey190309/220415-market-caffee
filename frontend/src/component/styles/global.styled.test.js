import React from 'react'
import renderer from 'react-test-renderer'

import { MainContainer, MainItem } from './global.styled'

describe('global.styled', () => {
  describe('MainContainer', () => {
    test('Snapshot', () => {
      const tree = renderer.create(<MainContainer />).toJSON()
      expect(tree).toMatchSnapshot();
      // console.log('global.styled ->', tree)
      expect(tree).toHaveStyleRule(
        'display', 'grid',
        'grid-template-columns', 'auto'
      )
     })
  })
  describe('MainItem', () => {

  });
});