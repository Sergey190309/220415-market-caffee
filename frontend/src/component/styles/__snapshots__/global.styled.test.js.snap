// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`global.styled MainContainer Snapshot 1`] = `
<div
  className="sc-bczRLJ dDhCsx"
/>
`;
